openassessment/assessment/models/ai.py

```diff
@@ -40,6 +40,35 @@
 )
 
 
+def essay_text_from_submission(submission):
+    """
+    Retrieve the submission text.
+
+    Submissions are arbitrary JSON-blobs, which *should*
+    contain a single key, "answer", containing the essay
+    submission text.
+    If not, though, assume we've been given the essay text
+    directly (convenient for testing).
+    """
+    if isinstance(submission, dict):
```

Comment:
I just moved this code from below to a separate function. I get your point, but this will require updating a lot of test data.